### PR TITLE
[Feature][docs] Add committers to team page

### PR DESCRIPTION
**Pull request by wuchunfu**
[Feature][docs] Add committers to team page

https://github.com/apache/incubator-seatunnel/issues/1604

### CODE REVIEW

File: src/pages/team/languages.json [hunk 1]
```diff
@@ -64,53 +64,79 @@
...
+        "name": "Huajie Wang"
+      },
+      {
+        "apacheId": "wuchunfu",
+        "avatarUrl": "https://avatars.githubusercontent.com/u/29206593?v=4",
+        "email": "wuchunfu@apache.org",
+        "gitUrl": "https://github.com/wuchunfu",
+        "githubId": "wuchunfu",
+        "name": "ChunFu Wu"
+      }
```
Comment (zhongjiajie) [inline src/pages/team/languages.json:139]:
```suggestion
      },
      {
        "apacheId": "yangxiang",
        "avatarUrl": "https://avatars.githubusercontent.com/u/8814185?v=4",
        "email": "yangxiang@apache.org",
        "gitUrl": "https://github.com/yx91490",
        "githubId": "yx91490",
        "name": "Yang Xiang"
      }
```

Comment (zhongjiajie) [reply src/pages/team/languages.json:139]: We have a new committer @yx91490 . Please make sure your information is correct.

Comment (zhongjiajie) [reply src/pages/team/languages.json:139]: BTW, we can see all our committer and PPMC mentor in https://whimsy.apache.org/roster/

File: src/pages/team/languages.json [hunk 2]
```diff
@@ -6,55 +6,55 @@
     },
     "mentor": [
       {
-        "apacheId": "",
+        "apacheId": "ningjiang",
         "avatarUrl": "https://avatars.githubusercontent.com/u/219644?v=4",
-        "email": "willem.jiang@gmail.com",
+        "email": "kezhenxu94@apache.org",
```
Comment (zhongjiajie) [inline src/pages/team/languages.json:11]:
```suggestion
        "email": "ningjiang@apache.org",
```

Comment (zhongjiajie) [reply src/pages/team/languages.json:11]: Some mistake here

File: src/pages/team/languages.json [hunk 3]
```diff
@@ -120,111 +146,137 @@
...
```
Comment (zhongjiajie) [inline src/pages/team/languages.json:151]:
```suggestion
        "email": "ningjiang@apache.org",
```

File: src/pages/team/languages.json [hunk 4]
```diff
@@ -120,111 +146,137 @@
...
+        "name": "Huajie Wang"
+      },
+      {
+        "apacheId": "wuchunfu",
+        "avatarUrl": "https://avatars.githubusercontent.com/u/29206593?v=4",
+        "email": "wuchunfu@apache.org",
+        "gitUrl": "https://github.com/wuchunfu",
+        "githubId": "wuchunfu",
+        "name": "ChunFu Wu"
+      }
```
Comment (zhongjiajie) [inline src/pages/team/languages.json:279]:
```suggestion
      },
      {
        "apacheId": "yangxiang",
        "avatarUrl": "https://avatars.githubusercontent.com/u/8814185?v=4",
        "email": "yangxiang@apache.org",
        "gitUrl": "https://github.com/yx91490",
        "githubId": "yx91490",
        "name": "Yang Xiang"
      }
```

**Review by zhongjiajie — COMMENTED** [review]
I add some suggestion here

File: src/pages/team/languages.json (hunk 2 above)
Comment (wuchunfu) [reply src/pages/team/languages.json:11]: @zhongjiajie Thanks for the reminder, I will fix it

File: src/pages/team/languages.json (hunk 4 above)
Comment (wuchunfu) [reply src/pages/team/languages.json:279]: @zhongjiajie Thanks for the reminder, I will add it

File: src/pages/team/languages.json (hunk 1 above)
Comment (wuchunfu) [reply src/pages/team/languages.json:139]: @zhongjiajie Thanks for the reminder.

**Review by zhongjiajie — APPROVED** [review]
LGTM, a great add

**Pull request merged**